### ISSUE TITLE
info site not up to date

### ISSUE THREAD

**Issue by sobvan** [pancakeswap/pancake-info#26]
Info site is not synced to the latest state of the blockchain. Will this ever catch up?

Also, I am looking for a way to track the value of my staked LP tokens. I have found none. Is it even possible?

**Comment by PiranhaUK** [comment]
Any ETA on a fix for this? 

It reflects poorly on the project that this is broken as its difficult for users to correctly estimate the liquidity/volume of each pair.

**Comment by pmaniora** [comment]
maybe https://github.com/pancakeswap/pancake-info/issues/24#issuecomment-783406838 could be a hint to resolve this.
As i checked out, subgraph is syncing, but the speed is to slow, as i calculated if it keep syncing with speed as now it will take over 8 days.

**Comment by PiranhaUK** [comment]
Is it not this code commit - now using a fixed time?

https://github.com/pancakeswap/pancake-info/pull/23/commits/6612b3157ca464b3a9cfc9ab6e132ea273dcedd6

**Comment by pmaniora** [comment]
@PiranhaUK no, this is used to not ask GraphQL for block that is not synced which gives an error and not displaying charts

**Comment by gotjoshua** [comment]
any ETA for a fix?  why don't you change your config to query a different node that is actually up to date ?

**Comment by bendrexl** [comment]
I'm new to the site, but it looks pretty bad to have this broken since March 17. Has this been acknowledged?

**Comment by pmaniora** [comment]
As I understand it's not trivial problem, pancakeswap offers 150k $ bounty for fixing this:
https://pancakeswap.medium.com/pancakeswap-launches-150-000-info-site-bounty-cff20ec9aff0

**Comment by zaibon** [comment]
Does this issue means, that today, there is not tools for liquidity provider to track they performance of the pool they invest in ?
Also the issue mention that "The graph" is too slow to be fully sync with the blockchain, but the info page is not just out of sync, it does not sync any more block at all. So this service is pretty much dead. Am I right or did I miss something ?
